.github/workflows/build.yml

```diff
@@ -86,6 +117,7 @@ jobs:
       id: get_version
       run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
 
+    # TODO: This action is not maintained - migrate to suitable replacement
```

Comment:
We can use this instead: https://github.com/softprops/action-gh-release

Reply:
Yup.  Already had my mind on that - for a future PR.